piptools/writer.py

```diff
@@ -116,10 +116,10 @@ def write_header(self) -> Iterator[str]:
         if self.emit_header:
             yield comment("#")
             yield comment(
-                "# This file is autogenerated by pip-compile with python "
-                f"{sys.version_info.major}.{sys.version_info.minor}"
+                "# This file is autogenerated by pip-compile with Python "
+                f"{sys.version_info.major}.{sys.version_info.minor}."
             )
-            yield comment("# To update, run:")
+            yield comment("# To update it, run:")
```

Comment:
"Python" with capital P is good.

This change is also fine but I think the original is fine too and reads a bit better:

```suggestion
            yield comment("# To update, run:")
```